src/emrun_prejs.js

```diff
@@ -1,7 +1,8 @@
-// Copyright 2013 The Emscripten Authors.  All rights reserved.
-// Emscripten is available under two separate licenses, the MIT license and the
-// University of Illinois/NCSA Open Source License.  Both these licenses can be
-// found in the LICENSE file.
+/**
+ * @license
+ * Copyright 2020 Emscripten authors
```

Comment:
What about preserving those dates?

Reply:
How about capital `A` in `authors` to match the convention uses here and in chromium.  Also the `The`?

Reply:
Happy to change to `Authors` with capitals.

Adding `The` seems unnecessary to me?

Changing the dates seems unnecessary to me, and I am actually modifying the files now, so the copyright would need to reflect that, I think? But IANAL and maybe you know more?

Reply:
Asking some people, the answer is to preserve the year. Changed the PR for that now.